README.md

```diff
@@ -98,15 +98,131 @@ This release extends support for RHEL v8.5.
+| gfx906:xnack- | Radeon VII | Vega 20 XT      | AMD Radeon™ PRO VII                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx908:xnack- | MI100      | MI100 XL        | AMD Instinct MI100                                |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ✅    |
+| gfx90a:xnack+ | MI210      | Aldebaran       | TBA                                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ✅    |
+| gfx90a:xnack- | MI210      | Aldebaran       | TBA                                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ✅    |
+| gfx1010       | Navi       | Navi 10 XT      | AMD Radeon™ PRO 5600M                             |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1010       | Navi       | Navi 10 XLE     | AMD Radeon™ PRO 5600 XT                           |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1010       | Navi       | Navi 10 XL      | AMD Radeon™ RX 5700                               |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1010       | Navi       | Navi 10 XT      | AMD Radeon™ RX 5700 XT                            |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1012       | Navi       | Navi 14 XT      | AMD Radeon™ RX 5500                               |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1012       | Navi       | Navi 14 XTX     | AMD Radeon™ RX 5500 XT                            |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ RX 6800                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ RX 6800 XT                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ RX 6900 XT                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ RX 6950 XT                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
```

Comment:
I'm not sure AMD ever called this "Big Navi", so I'm sure we should use that terminology in the README.

Reply:
Yes, you are right. "Big Navi" is only a nickname: https://en.wikipedia.org/wiki/Radeon_RX_6000_series - I reworked the file to call them "Navi 2X" based on the same reference in my latest commit.